spec/models/assignment_spec.rb

```diff
@@ -214,4 +214,124 @@ def create_pending_responding_assignment(kase, manager)
                         user_id: manager.id)
     end
   end
+
+  context 'marking case as dirty' do
+
+    let(:kase)    { create :case, :clean }
+
+    context 'responding assignment' do
+      context 'new assignment' do
+        it 'marks the case as dirty' do
+          create :assignment, :responding, case_id: kase.id
+          expect(kase.reload).to be_dirty
+        end
+
+        it 'queues the job' do
+          expect {
+            create :assignment, :responding, case_id: kase.id
+          }.to have_enqueued_job(SearchIndexUpdaterJob).at_least(1)
```

Comment:
Can we check that it enqueues the job with a 10s delay?

Reply:
done